### PR TITLE
configure grpc-gateway service via yaml file

### PR DESCRIPTION
**Pull request by jkralik**

### CODE REVIEW

File: grpc-gateway/config.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    address: "0.0.0.0:9100"
+    tls:
+      caPool: "/secrets/public/rootca.crt"
+      keyFile: "/secrets/private/cert.key"
+      certFile: "/secrets/public/cert.crt"
+      clientCertificateRequired: true
+    authorization:
+      authority: ""
+      audience: ""
+      ownerClaim: sub
```
Comment (ondrejtomcik) [inline grpc-gateway/config.yaml]: Forgot, why do we have it here as it is relevant only for authorization-server which parses the token?

File: resource-directory/config.yaml [hunk 2]
```diff
@@ -25,6 +25,8 @@ apis:
           useSystemCAPool: false
 clients:
   eventBus:
+    # number of routines to process events in projection
+    goPoolSize: 16
```
Comment (ondrejtomcik) [inline resource-directory/config.yaml:28]: Why you added it only to RD and not RA ?
Please update also documentation Trishia already did for RD. It's in v2.

File: resource-directory/config.yaml [hunk 3]
```diff
@@ -35,8 +37,6 @@ clients:
   eventStore:
     # expiration time of cached resource in projection
     cacheExpiration: 1m
-    # number of routines to process events in projection
-    goPoolSize: 16
```
Comment (ondrejtomcik) [inline resource-directory/config.yaml:39]: And this remove from doc.

**Review by ondrejtomcik — CHANGES_REQUESTED** [review]

File: grpc-gateway/config.yaml (hunk 1 above)
Comment (jkralik) [reply grpc-gateway/config.yaml]: removed

File: resource-directory/config.yaml (hunk 2 above)
Comment (jkralik) [reply resource-directory/config.yaml:28]: updated doc - new evenstore doesn't use goPool.

File: resource-directory/config.yaml (hunk 3 above)
Comment (jkralik) [reply resource-directory/config.yaml:39]: updated

File: authorization/config.yaml [hunk 4]
```diff
@@ -32,6 +31,7 @@ apis:
       clientCertificateRequired: true
 oauthClients:
   device:
+    ownerClaim: sub
```
Comment (ondrejtomcik) [inline authorization/config.yaml]: It doesn't belong to oauthClients.device. When we move this configuration to coapgateway, you will have to add it to other place in this authorization-server config - again breaking change. And, coap-gateway doesn't need it anyway. It's the general configuration of the authorization server.

CoAP gateway just sends "register device" request to authorization server with token, it won't care about jwt claims.

**Review by ondrejtomcik — CHANGES_REQUESTED** [review]

File: authorization/config.yaml (hunk 4 above)
Comment (ondrejtomcik) [reply authorization/config.yaml]: As discussed, will be moved to storage group.

**Review by ondrejtomcik — APPROVED** [review]

**Pull request merged**